data-raw/geocode-data/geocode.r

```diff
@@ -204,17 +217,22 @@ processFile <- function(inputFilePath, outputFilePath, email)
 #'
 #' Interactive mode will prompt the user for input files and email.
 executeInteractiveScript <- function() {
-  email = readline(
-    "Please provide your email. We need this to comply with terms of service for the nominatim geocoder). "
+  cat(
+    "Please provide your email.\n",
+    "We need this to comply with terms of service for the nominatim geocoder).",
+    sep = ""
   )
+  email <- readline("Email: ")
   
-  downloadDryadData = readline(
-    "Do you want to download data from Dryad Data? Y = download the data directly from datadryad.org, N = enter your own file path. "
+  cat(
+    "Do you want to download data from Dryad Data?\n",
+    "Y = download the data directly from datadryad.org,\n",
+    "N = enter your own file path.",
+    sep = ""
   )
+  downloadDryadData <- readline("Downlead? ")
```

Comment:
Download is misspelled

Reply:
Is it correct etiquette for me to "resolve" the conversation once i've pushed corrections?